lib/active_merchant/billing/gateways/worldpay.rb

```diff
@@ -778,6 +778,7 @@ def add_shopper(xml, options)
             xml.acceptHeader options[:accept_header]
             xml.userAgentHeader options[:user_agent]
           end
+          add_shopper_id(xml, options, false)
```

Comment:
I actually don't think we should add add_shopper_id here but in [add_network_tokenization_card](https://github.com/activemerchant/active_merchant/blob/master/lib/active_merchant/billing/gateways/worldpay.rb#L651).

Reply:
Done!